posthog/hogql_queries/web_analytics/stats_table.py

```diff
@@ -564,7 +564,8 @@ def where_breakdown(self):
                 return parse_expr("tupleElement(breakdown_value, 2) IS NOT NULL")
             case WebStatsBreakdown.VIEWPORT:
                 return parse_expr(
-                    "tupleElement(breakdown_value, 1) IS NOT NULL AND tupleElement(breakdown_value, 2) IS NOT NULL"
+                    "tupleElement(breakdown_value, 1) IS NOT NULL AND tupleElement(breakdown_value, 2) IS NOT NULL AND "
+                    "tupleElement(breakdown_value, 1) != 0 AND tupleElement(breakdown_value, 2) != 0"
```

Comment:
nitpicking you can probably do something like `ifNull(thing, 0) > 0` 
but without testing what's faster it's just noise :)